gym_electric_motor/physical_systems/electric_motors/externally_excited_synchronous_motor.py

```diff
@@ -122,20 +124,28 @@ class ExternallyExcitedSynchronousMotor(SynchronousMotor):
-            [                     0,                                           0, -mp['r_s'],                                           0,                                0,   1,                                0, -mp['l_d'] * mp['p'],                                                      0, -mp['p'] * mp['l_m']],
-            [                     0, mp['l_m'] * mp['r_s'] / (sigma * mp['l_d']),          0,                          -mp['r_e'] / sigma, -mp['l_m'] / (sigma * mp['l_d']),   0,                        1 / sigma,                    0, -mp['p'] * mp['l_m'] * mp['l_q'] / (sigma * mp['l_d']),                    0],
-            [               mp['p'],                                           0,          0,                                           0,                                0,   0,                                0,                    0,                                                      0,                    0],
+            #                 omega,                                               i_d,        i_q,                                                              i_e,                              u_d, u_q,                                                    u_e,          omega * i_d,                                                  omega * i_q,                         omega * i_e
+            [                     0,                          -mp['r_s'] / mp['sigma'],          0, mp['l_M'] * mp['r_E'] / (mp['sigma'] * mp['l_E']) * mp['i_k_rs'],                        1 / mp['sigma'],   0, -mp['l_M'] * mp['k'] / (mp['sigma'] * mp['l_E']),                    0,                            mp['l_q'] * mp['p'] / mp['sigma'],                                   0],
+            [                     0,                                                 0, -mp['r_s'],                                                                0,                                      0,   1,                                                0, -mp['l_d'] * mp['p'],                                                            0, -mp['p'] * mp['l_M'] * mp['i_k_rs']],
+            [                     0, mp['l_M'] * mp['r_s'] / (mp['sigma'] * mp['l_d']),          0,                          -mp['r_E'] / mp['sigma'] * mp['i_k_rs'], -mp['l_M'] / (mp['sigma'] * mp['l_d']),   0,                            mp['k'] / mp['sigma'],                    0, -mp['p'] * mp['l_M'] * mp['l_q'] / (mp['sigma'] * mp['l_d']),                                   0],
+            [               mp['p'],                                                 0,          0,                                                                0,                                      0,   0,                                                0,                    0,                                                            0,                                   0],
         ])
 
```

Comment:
When I use the linear model, which Mario provide to us, the matrix should look like:
`self._model_constants = np.array([
            #                 omega,                                                           i_d,                      i_q,                                                                          i_e,                                                u_d,             u_q,                                                          u_e,                        omega * i_d,                                                              omega * i_q,                                       omega * i_e
            [                     0,                        -mp['r_s'] / (mp['sigma'] * mp['l_d']),                        0, mp['l_M'] * mp['r_E'] / (mp['sigma'] * mp['l_E'] * mp['l_d']) * mp['i_k_rs'],                      1 / (mp['sigma'] * mp['l_d']),               0, -mp['l_M'] * mp['k'] / (mp['sigma'] * mp['l_E'] * mp['l_d']),                                  0,                          mp['l_q'] * mp['p'] / (mp['sigma'] * mp['l_d']),                                                 0],
            [                     0,                                                             0, -mp['r_s'] / (mp['l_q']),                                                                            0,                                                  0, 1 / (mp['l_q']),                                                            0, -mp['l_d'] * mp['p'] / (mp['l_q']),                                                                        0, -mp['p'] * mp['l_M'] * mp['i_k_rs'] / (mp['l_q'])],
            [                     0, mp['l_M'] * mp['r_s'] / (mp['sigma'] * mp['l_d'] * mp['l_E']),                        0,                        -mp['r_E'] / (mp['sigma'] * mp['l_E']) * mp['i_k_rs'], -mp['l_M'] / (mp['sigma'] * mp['l_d'] * mp['l_E']),               0,                          mp['k'] / (mp['sigma'] * mp['l_E']),                                  0, -mp['p'] * mp['l_M'] * mp['l_q'] / (mp['sigma'] * mp['l_d'] * mp['l_E']),                                                 0],
            [               mp['p'],                                                             0,                        0,                                                                            0,                                                  0,               0,                                                            0,                                  0,                                                                        0,                                                 0],
        ]) 
`
to my opinion.